operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1194,6 +1194,17 @@ spec:
               strimzi.io/kind: cluster-operator
           template:
             metadata:
+              annotations:
+                 operator-image: registry.redhat.io/amq-streams/strimzi-rhel9-operator@sha256:95f5aa75cd1f7228e78fd4d88d786713fba4cf828dc22bc2dd1d0380909c1aef
+                 kafka-images: |
+                   3.6.0=registry.redhat.io/amq-streams/kafka-36-rhel9@sha256:177484ebf6f663eedfc558285e4e89e817277389250aec29323452082b6949e4
```

Comment:
Did you double check that https://github.com/containerbuildsystem/atomic-reactor/commit/a8294d0e862e1465b95071af0e0722b2c21a8328 is going to handle our version=<pullspec>values okay?  Looking at the code it should but I haven't run it.

Reply:
We did the same thing for RHOSAK, and even had an annotation with JSON where the image references were string values within.

Reply:
It is preserved when running through OSBS and gets expanded correctly when tested locally. The other annotations, kafka-current-image and kafka-previous-image, are used by OSBS to created the `relatedImages` section